### PR TITLE
HLR | Replace wizard with subtask

### PR DESCRIPTION
**Pull request by Mottie**
## Summary

- _(Summarize the changes that have been made to the platform)_
  > The wizard (on the /start page) is a deprecated pattern. We're updating it to the subtask pattern that shows one question per page. This work also include some code cleanup in Supplemental Claims because it also uses a subtask and making the code similar makes it easier to read and compare.
- _(If bug, how to reproduce)_
- _(What is the solution, why is this the solution)_
  > Copy subtask pattern from Supplemental Claims & remove wizard code, constants and tests.
- _(Which team do you work for, does your team own the maintenance of this component?)_
  > Benefits Decision Review
- _(If using a flipper, what is the end date of the flipper being required/success criteria being targeted)_

## Related issue(s)

[#57821](https://github.com/department-of-veterans-affairs/va.gov-team/issues/57821)

## Testing done

- _Describe what the old behavior was prior to the change_
- _Describe the steps required to verify your changes are working as expected_
- _Describe the tests completed and the results_
- _Exclusively stating 'Specs and automated tests passing' is NOT acceptable as appropriate testing
- _Optionally, provide a link to your [test plan](https://depo-platform-documentation.scrollhelp.site/developer-docs/create-a-test-plan-in-testrail) and [test execution records](https://depo-platform-documentation.scrollhelp.site/developer-docs/execute-tests-in-testrail)_

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|  | Wizard (before)  | Subtask (after) |
| ------- | ------ | ------ |
| Start page | hlr wizard showing a radio pair. When you select a claim other than disability compensation, it expands down with additional information | hlr subtask with the same radio pair, but includes a continue button below the radios. After making the claim other than disability compensation, a new page is rendered - on the same url - with a back button |

## What areas of the site does it impact?

Higher-Level Review

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [x] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

**Review by powellkerry — DISMISSED** [review]
Looks good once CI passes

File: src/applications/appeals/996/tests/hlr-keyboard-only.cypress.spec.js [hunk 1]
```diff
@@ -35,12 +37,12 @@ describe('Higher-Level Review keyboard only navigation', () => {
...
 
-      // Wizard
+      // Subtask
       cy.url().should('include', '/start');
-      cy.tabToElement('input[value="compensation"]');
+      cy.tabToElement('input#compensationinput'); // ID of va-radio-option input
       cy.realPress('Space');
 
-      cy.tabToElement('.vads-c-action-link--green');
+      cy.tabToElement('va-button');
```
Comment (shaunburdick) [inline src/applications/appeals/996/tests/hlr-keyboard-only.cypress.spec.js:45]: Could you add the containing class to this selector?

Comment (Mottie) [reply src/applications/appeals/996/tests/hlr-keyboard-only.cypress.spec.js]: Done

**Review by anniebtran — APPROVED** [review]
Did a live review with Robin and Shaun. LGTM!

**Pull request merged**